DevCycle.SDK.Server.Cloud.MSTests/DVCTest.cs

```diff
@@ -35,6 +35,8 @@ public void GetFeaturesTest()
             Assert.IsNotNull(result);
             Assert.AreEqual(1, result.Count);
             Assert.IsNotNull(result["show-feature-history"]);
+            Assert.IsNotNull(result["show-feature-history"].VariationKey);
```

Comment:
Maybe also add a check for is not empty string too

Reply:
done!